src/content/docs/en/pages/devtools/cli/azion-cli/commands/init/init.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Azion CLI init
+description: Initialize a new application from Azion's templates.
+meta_tags: cli, next, javascript, astro, vue, react, edge computing
+namespace: documentation_cli_command_init
+menu_namespace: cliMenuAlpha
+permalink: /documentation/devtools/cli/init/
+---
+
+Initialize an edge application from a starter template. You can choose a template based on Next, Vue, React, Angular, Astro, and other frameworks.
+
+:::note 
+In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
```

Comment:
```suggestion
In case one or more required fields aren't informed through the specific flags, an interactive prompt asks for the missing information.
```